src/pages/index.js

```diff
@@ -119,17 +214,36 @@ function handleEscapePress(evt) {
   }
 }
 
-
 function handleAddCardSubmit(evt) {
   evt.preventDefault(settings);
   console.log(editModalNameInput.value);
   console.log(editModalDescriptionInput.value);
-  const inputValues = { name: cardCaptionInput.value, link: cardLinkInput.value };
-  const cardEl = getCardElement(inputValues);
-  cardsList.prepend(cardEl);
-  evt.target.reset();
-  disableButton(cardSubmitBtn, settings);
-  closeModal(cardModal, settings);
+  const inputValues = {
+    name: cardCaptionInput.value,
+    link: cardLinkInput.value,
+  };
+
+  api.addCard(inputValues).then(() => {
+    const cardEl = getCardElement(inputValues);
+    cardsList.prepend(cardEl);
+    evt.target.reset();
+    disableButton(cardSubmitBtn, settings);
+    closeModal(cardModal, settings);
+  });
+}
+
+function handleAvatarSubmit(evt) {
+  evt.preventDefault(settings);
```

Comment:
*  According to the task, you need to change submit buttons text (`Saving…`) while waiting for the server response. And the user should see it.